dags/ethereumetl_airflow/build_parse_v2_dag.py

```diff
@@ -0,0 +1,216 @@
+
+            client = bigquery.Client()
+
+            parse_dataset_folder(
+                bigquery_client=client,
+                dataset_folder=dataset_folder,
+                ds=ds,
+                source_project_id=SOURCE_PROJECT_ID,
+                source_dataset_name=SOURCE_DATASET_NAME,
+                destination_project_id=parse_destination_dataset_project_id,
+                sqls_folder=os.path.join(dags_folder, 'resources/stages/parse/sqls'),
+                parse_all_partitions=parse_all_partitions
+            )
+
+        dataset_name = get_dataset_name(dataset_folder)
+        parsing_operator = PythonOperator(
+            task_id=f'parse_tables_{dataset_name}',
+            python_callable=parse_v2_task,
+            execution_timeout=timedelta(minutes=60 * 4),
+            dag=dag
+        )
+
+        return parsing_operator
+
+    def create_add_view_task(dataset_name, view_name, sql):
+        def create_view_task(ds, **kwargs):
+            client = bigquery.Client()
+
+            dest_table_name = view_name
+            dest_table_ref = create_dataset(client, dataset_name, parse_destination_dataset_project_id).table(dest_table_name)
+
+            print('View sql: \n' + sql)
+
+            create_view(client, sql, dest_table_ref)
+
+        create_view_operator = PythonOperator(
+            task_id=f'create_view_{view_name}',
+            python_callable=create_view_task,
+            execution_timeout=timedelta(minutes=10),
+            dag=dag
+        )
+
+        return create_view_operator
+
+    def create_share_dataset_task(dataset_name):
+        def share_dataset_task(**kwargs):
+            if parse_destination_dataset_project_id != 'blockchain-etl':
+                logging.info('Skipping sharing dataset.')
+            else:
+                client = bigquery.Client()
+                share_dataset_all_users_read(client, f'{parse_destination_dataset_project_id}.{dataset_name}')
+                share_dataset_all_users_read(client, f'{parse_destination_dataset_project_id}-internal.{dataset_name}')
+
+        share_dataset_operator = PythonOperator(
+            task_id='share_dataset',
+            python_callable=share_dataset_task,
+            execution_timeout=timedelta(minutes=10),
+            dag=dag
+        )
+
+        return share_dataset_operator
+
+    wait_for_ethereum_load_dag_task = ExternalTaskSensor(
+        task_id='wait_for_ethereum_partition_dag',
+        external_dag_id=PARTITION_DAG_ID,
+        external_task_id='done',
+        execution_delta=timedelta(minutes=30),
+        priority_weight=0,
+        mode='reschedule',
+        poke_interval=5 * 60,
+        timeout=60 * 60 * 12,
+        dag=dag)
+
+    parse_v2_task = create_parse_v2_task()
+    wait_for_ethereum_load_dag_task >> parse_v2_task
+
+    checkpoint_task = BashOperator(
+        task_id='parse_all_checkpoint',
+        bash_command='echo parse_all_checkpoint',
+        priority_weight=1000,
```

Comment:
Think this isn't desirable now

Reply:
Actually thinking we should try omitting this even on parse_v1 -- just to see if this on its own produces an improvement in scheduler performance.